### ISSUE TITLE
Translate it into Chinese

### ISSUE THREAD

**Issue by moreJs** [xgrommx/rx-book#65]
Firstly, this book is amazing, so important to understand RxJx。

 But, there are not too many Chinese books on RxJx. So I want tanslate it into Chinese, Can I do it ?

 Thanks a lot!

**Comment by moreJs** [comment]
@xgrommx

**Comment by moreJs** [comment]
@xgrommx you can create a Crowdin Project just like this (https://github.com/thejameskyle/babel-handbook/blob/master/CONTRIBUTING.md)

**Comment by xgrommx** [comment]
Yes, I'll make a preparation for multilanguages. I'll notify you soon. But I don't know Chinese =)

**Comment by moreJs** [comment]
@xgrommx 
Haha。
Maybe after this project, you will be good at  Chinese .

**Comment by deyuhua** [comment]
This book also teach me a lot about RP, so i want to join you!
@moreJs @xgrommx

**Comment by moreJs** [comment]
@deyuhua  https://github.com/RxJS-CN
